packages/venia-ui/lib/RootComponents/CMS/cms.js

```diff
@@ -35,7 +35,17 @@ const CMSPage = props => {
             data.cmsPage.content.length > 0 &&
             !data.cmsPage.content.includes('CMS homepage content goes here.')
         ) {
-            content = <RichContent html={data.cmsPage.content} />;
+            content = (
+                <>
```

Comment:
Happy to make this update, but would just like to point out that the only reason I used the shorthand here is because I thought Magento were preferring it, as it's used in many files in PWA Studio. A simple Find In Files in the codebase will find 8 other places that the shorthand is used, including the very file that I've edited

Reply:
FWIW, I started out liking `React.Fragment` but found a typo broke the whole build with a non-helpful error. We ended up switcing everything to `Fragment` destructured because that error is dev-traceable. 

And we specifically avoided `<>` because at least one major editor (Atom) doesn't understand how to auto-close tags with no node-name, which is frustrating.

Reply:
Sure thing! I totally agree, I much prefer seeing `<Fragment>` rather than the _magic_ `<>`. I just wanted to point out the use of the shorthand because I'm being told that the full tag is preferred, when in the actual component I'm editing it's clearly not being used. If the decision has been made since this code was written, that's totally fine, but just need to make sure we're aware of these sort of things when making statements like X is preferred over Y